### PR TITLE
DAOS-11635 doc: fixes to QSG setup guide (#10582)

### PR DESCRIPTION
**Pull request by Michael-Hennecke**
fixes to QSG setup guides (rhel and suse)

Doc-only: true
Signed-off-by: Michael Hennecke <michael.hennecke@intel.com>

### CODE REVIEW

File: docs/QSG/setup_suse.md [hunk 1]
```diff
@@ -318,11 +319,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/maste
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos0
-			scm_class: dcpm
-			scm_list: [/dev/pmem0]
-			bdev_class: nvme
-			bdev_list: ["0000:81:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /mnt/pmem0
```
Comment (kccain) [inline docs/QSG/setup_suse.md:325]: /mnt/pmem0 -> /mnt/daos0 to match PR 10582 for release/2.2

File: docs/QSG/setup_suse.md [hunk 2]
```diff
@@ -333,21 +339,26 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/maste
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos1
-			scm_class: dcpm
-			scm_list: [/dev/pmem1]
-			bdev_class: nvme
-			bdev_list: ["0000:83:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /mnt/pmem1
```
Comment (kccain) [inline docs/QSG/setup_suse.md:345]: /mnt/pmem1 -> /mnt/daos1 to match PR 10582 for release/2.2

**Review by kccain — CHANGES_REQUESTED** [review]

**Comment by johannlombardi** [comment]
@Michael-Hennecke You intend to update the PR based on Ken's feedback?

File: docs/QSG/setup_rhel.md [hunk 3]
```diff
@@ -301,11 +300,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/maste
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos0
-			scm_class: dcpm
-			scm_list: [/dev/pmem0]
-			bdev_class: nvme
-			bdev_list: ["0000:81:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /tmp/pmem0
```
[the remark this answers is not in the document]
Comment (Michael-Hennecke) [reply docs/QSG/setup_rhel.md]: done

File: docs/QSG/setup_suse.md (hunk 1 above)
Comment (Michael-Hennecke) [reply docs/QSG/setup_suse.md]: done

File: docs/QSG/setup_rhel.md [hunk 4]
```diff
@@ -316,11 +320,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/maste
...
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos1
-			scm_class: dcpm
-			scm_list: [/dev/pmem1]
-			bdev_class: nvme
-			bdev_list: ["0000:83:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /tmp/pmem1
```
[the remark this answers is not in the document]
Comment (Michael-Hennecke) [reply docs/QSG/setup_rhel.md]: done

File: docs/QSG/setup_suse.md (hunk 1 above)
Comment (Michael-Hennecke) [reply docs/QSG/setup_suse.md]: done

File: docs/QSG/setup_suse.md (hunk 2 above)
Comment (Michael-Hennecke) [reply docs/QSG/setup_suse.md]: done

**Review by kccain — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**